### PR TITLE
Upgrade to appium java_client_5

### PR DESCRIPTION
**Pull request by fuzzybinary**
Also improved unit tests by splitting out a lot of the swipe tests.

### CODE REVIEW

File: src/test/java/com/joss/conductor/mobile/LocomotiveTest.java [hunk 1]
```diff
@@ -250,55 +258,178 @@ public void test_get_center_window() {
...
+        Map<String, List<Object>> expectedParameters = getTouchActionParameters(expected);
+
+        assertThat((Map<String, List<Object>>)actualParameters, matchesEntriesIn((Map<String, List<Object>>)expectedParameters));
+    }
+
+    private void initMockDriver() {
+        initMockDriver(null);
+    }
+
+    private void initMockDriver(WebElement mockElement) {
```
Comment (jossjacobo) [inline src/test/java/com/joss/conductor/mobile/LocomotiveTest.java]: The method name is ambiguous, it implies that this method is initializes _all_ of the mockDriver mocks when it only adds window dimension and (if element not `@null`) size mocks.

Comment (fuzzybinary) [reply src/test/java/com/joss/conductor/mobile/LocomotiveTest.java]: True, I'll adjust it.

File: src/test/java/com/joss/conductor/mobile/MockTestUtil.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import static org.hamcrest.Matchers.hasEntry;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import io.appium.java_client.TouchAction;
+import org.hamcrest.Matcher;
+
+public class MockTestUtil {
```
[the remark this answers is not in the document]
Comment (fuzzybinary) [reply src/test/java/com/joss/conductor/mobile/MockTestUtil.java]: That's exactly what it's for. Obviously comments are not needed ;)

File: pom.xml [hunk 3]
```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>io.appium</groupId>
             <artifactId>java-client</artifactId>
-            <version>4.1.2</version>
+            <version>5.0.0</version>
```
Comment (konwinkler) [inline pom.xml]: Could we go straight to latest bugfix version `5.0.4`? https://mvnrepository.com/artifact/io.appium/java-client/5.0.4

**Comment by konwinkler** [comment]
I'm excited to go to the latest Appium version. Thanks for updating all the swipe methods!

File: pom.xml (hunk 3 above)
Comment (fuzzybinary) [reply pom.xml]: Maybe? I'll check. (For some reason I feel like that didn't exist yesturday?)

**Comment by fuzzybinary** [comment]
Updated with requested changes.

**Pull request merged**